app/validators/blocks/list_collector_validator.py

```diff
@@ -6,82 +6,59 @@
 
 class ListCollectorValidator(BlockValidator, ValidateListCollectorQuestionsMixin):
     LIST_COLLECTOR_KEY_MISSING = "Missing key in ListCollector"
-    ADD_ANSWER_REFERENCE_NOT_IN_MAIN_BLOCK = (
-        "add_answer reference uses id not found in main block question"
-    )
+    REDIRECT_TO_ADD_BLOCK = "RedirectToAddBlock"
+    REMOVE_LIST_ITEM_AND_ANSWERS = "RemoveListItemAndAnswers"
+
     NO_RADIO_FOR_LIST_COLLECTOR = (
         "The list collector block does not contain a Radio answer type"
     )
+
     NO_RADIO_FOR_LIST_COLLECTOR_REMOVE = (
         "The list collector remove block does not contain a Radio answer type"
     )
-    NON_EXISTENT_LIST_COLLECTOR_ADD_ANSWER_VALUE = (
-        "The list collector block has an add_answer_value that is not "
-        "present in the answer values"
-    )
-    NON_EXISTENT_LIST_COLLECTOR_REMOVE_ANSWER_VALUE = "The list collector block has a remove_answer_value that is not present in the answer values"
-    REMOVE_ANSWER_REFERENCE_NOT_IN_REMOVE_BLOCK = (
-        "remove_answer reference uses id not found in remove_block"
-    )
+
+    NO_REDIRECT_TO_ADD_BLOCK = "RedirectToAddBlock action not found"
+
+    NO_REMOVE_LIST_ITEM_AND_ANSWERS = "RemoveListItemAndAnswers action not found"
```

Comment:
Things like these could reference the variable itself. Saves renaming if it were to change.
eg:
```py
    NO_REMOVE_LIST_ITEM_AND_ANSWERS = f"{REMOVE_LIST_ITEM_AND_ANSWERS} action not found"
```